cfgov/unprocessed/js/routes/common.js

```diff
@@ -8,6 +8,7 @@
 require( 'jquery' );
 require( 'jquery-easing' );
 require( 'cf-expandables' );
+require( '../modules/util/modernizr');
```

Comment:
Sorry to be a downer but this should really use `gulp-modernizr` and be part of the build flow instead of installing a custom modernizr build directly. If you'd rather that happens in another PR that's fine but please add a TODO.